epictrack-web/src/components/project/ProjectForm.tsx

```diff
@@ -267,6 +267,9 @@ export default function ProjectForm({ ...props }) {
             <Grid item xs={12}>
               <ETFormLabel required>Project Description</ETFormLabel>
               <TextField
+                defaultValue={
```

Comment:
would it work if we set it in the project.ts model, like the way we set the is_active

export const defaultProject = {
  is_active: true,
};